### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by bjfish**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Prior to submitting a PR, review the CONTRIBUTING.md document for recommendations on how to test:
+https://github.com/wasmerio/wasmer/blob/master/CONTRIBUTING.md#pull-requests
+-->
+
+# Description
+<!-- 
+Provide details regarding the change including motivation,
+links to related issues, and the context of the PR.
+-->
+
```
Comment (MarkMcCaskey) [inline .github/pull_request_template.md]: We should mention updating the changelog!

Comment (bjfish) [reply .github/pull_request_template.md]: I added a comment about updating the changelog.

**Comment by MarkMcCaskey** [comment]
Maybe the part about updating the changelog should be a checkbox outside of the comment.  Otherwise, I think most people will miss it

**Comment by bjfish** [comment]
@MarkMcCaskey I made the changelog comment a checkbox

**Pull request merged**